backend/game/cards/discard_cards/victory_point_manipulation_cards/drop_from_high_altitude.py

```diff
@@ -0,0 +1,16 @@
+from game.cards.discard_card import DiscardCard
+from game.values.locations import Locations
+import game.turn_actions.player_movement as player_movement
+
+
+class DropFromHighAltitude(DiscardCard):
+    def __init__(self):
+        super().__init__("Drop from High Altitude", 5,
+                         "+ 2[Star] and take control of Tokyo if you don't already control it")
```

Comment:
Drop from high altitude has a footnote: "(player choose to take control either City & Bay if both are occupied) "

This also means you shouldn't be moving everyone out of tokyo

Reply:
Right now we only have 1 Tokyo, the action i see is a force field would have to happen

Reply:
To be improved in future issue